### PR TITLE
App Home  Page  with placename cards with its cover photo added.

### PR DESCRIPTION
**Pull request by shashank3110**
Removed the default getphotos functions as we do not need them on home screen. Instead converted the horizontal scroll of categories to vertical cards and with cover photos with getcoverphotos function.

### CODE REVIEW

File: lib/home.dart [hunk 1]
```diff
@@ -139,29 +186,31 @@ class CategoriesTile extends StatelessWidget {
...
+                        // imageUrl: "https://images.pexels.com/photos/704320/pexels-photo-704320.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500",
+                              height: deviceHeight*hfactor,
+                              width: deviceWidth*wfactor,
                               fit: BoxFit.cover,
                             )),
                   Container(
-                    height: 50,
-                    width: 100,
+                    height: deviceHeight*hfactor,
+                    width: deviceWidth*wfactor,
```
Comment (dhirensr) [inline lib/home.dart:205]: looks good , would be handled for each device

File: lib/home.dart [hunk 2]
```diff
@@ -83,6 +115,10 @@ class _HomeState extends State<Home> {
 
   @override
   Widget build(BuildContext context) {
+    double deviceWidth = MediaQuery.of(context).size.width;
+    double deviceHeight = MediaQuery.of(context).size.height;
+    categoryPhotos.sort((a,b) => a.compareTo(b));
```
Comment (dhirensr) [inline lib/home.dart:120]: why such complicated logic?

Comment (shashank3110) [reply lib/home.dart:119]: first two lines are fines the sort one is not necessary if we use a map .

Comment (dhirensr) [reply lib/home.dart:119]: I think we can use a map , because code readability gets compromised with this logic. what do you think?

**Comment by shashank3110** [comment]
Missing brackets resolved.

**Pull request closed without merging**